bittensor/utils/balance.py

```diff
@@ -230,9 +230,9 @@ def __abs__(self):
     @staticmethod
     def from_float(amount: float):
         """
-        Given tao (float), return Balance object with rao(int) and tao(float), where rao = int(tao*pow(10,9))
+        Given tao, return :func:`Balance` object with rao(``int``) and tao(``float``), where rao = int(tao*pow(10,9))
```

Comment:
is the use of the quotes inside a docstring really necessary?

Reply:
I see. `:func:''Balance''` creates anchor link as `#` in HTML. I did it based on what I've seen Raj do. He will ultimately do it the way he wants users to see it :)